apis/nucleus/src/components/listbox/ListBoxRowColumn.jsx

```diff
@@ -111,36 +140,42 @@ const useStyles = makeStyles((theme) => ({
   },
 
   // The icons container holding tick and lock, shown inside fields.
-  icon: {
+  [`& .${classes.icon}`]: {
     display: 'flex',
     padding: theme.spacing(1, 1, 1, 0),
   },
 
   // Selection styles (S=Selected, XS=ExcludedSelected, A=Available, X=Excluded).
-  S: {
+  [`& .${classes.S}`]: {
     ...getSelectedStyle({ theme }),
   },
-  XS: {
+
+  [`& .${classes.XS}`]: {
     ...getSelectedStyle({ theme }),
     background: theme.palette.selected.excluded,
     color: theme.palette.selected.mainContrastText,
   },
-  A: {
+
+  [`& .${classes.A}`]: {
     background: theme.palette.selected.alternative,
     color: theme.palette.selected.alternativeContrastText,
   },
-  X: {
+
+  [`& .${classes.X}`]: {
     background: theme.palette.selected.excluded,
-    color: theme.palette.selected.mainContrastText,
+    color: theme.palette.selected.excludedContrastText,
```

Comment:
Is this change intentional? Using `excluded` instead of `main`

Reply:
yea it is. I think it was incorrect earlier, OR if the automatic codemod messed with it somehow. I tested to see what it looked like and this made more sense.